server/api/service_middleware_test.go

```diff
@@ -44,11 +45,11 @@ func (s *testServiceMiddlewareSuite) SetUpSuite(c *C) {
 	s.urlPrefix = fmt.Sprintf("%s%s/api/v1", addr, apiPrefix)
 }
 
-func (s *testServiceMiddlewareSuite) TearDownSuite(c *C) {
+func (s *testAuditMiddlewareSuite) TearDownSuite(c *C) {
 	s.cleanup()
 }
 
-func (s *testServiceMiddlewareSuite) TestConfigAudit(c *C) {
+func (s *testAuditMiddlewareSuite) TestConfigAuditSwitch(c *C) {
```

Comment:
Please add a test for the default value.

Reply:
update